package.json

```diff
@@ -25,7 +25,7 @@
     "release": "./scripts/travis/release.sh",
     "build": "gulp build",
     "test": "gulp test && node test/accessibility/wcag.js",
-    "cf-link": "cd ./src/cf-core && npm link && cd ../cf-buttons/ && npm link && cd ../cf-typography && npm link && cd ../cf-pagination && npm link"
+    "cf-link": "cd ./src/cf-core && npm link && cd ../cf-buttons/ && npm link && cd ../cf-typography && npm link && cd ../cf-pagination && npm link && npm link && cd ../cf-layout"
```

Comment:
npm linking cf-layout doesn't work, if you switch to `&& cd ../cf-layout && npm link` it should work!